Makefile

```diff
@@ -9,6 +9,7 @@ GOVET_IGNORE_RULES = \
 # List of executables needed for 'make generate'
 GENERATE_DEPENDENCIES = \
 	github.com/golang/mock/mockgen \
+	github.com/gogo/protobuf/protoc-gen-gogoslick \
```

Comment:
It's basically the same as Google's stuff https://github.com/gogo/protobuf/blob/master/LICENSE